browser/profiles/profile_util.cc

```diff
@@ -46,7 +46,8 @@ bool IsGuestProfile(content::BrowserContext* context) {
 
 bool IsTorDisabledForProfile(Profile* profile) {
 #if BUILDFLAG(ENABLE_TOR)
-  return TorProfileServiceFactory::IsTorDisabled() || profile->IsGuestSession();
+  return TorProfileServiceFactory::IsTorDisabled(profile) ||
+         profile->IsGuestSession();
```

Comment:
I wonder if `profile->IsGuestSession();` should now be moved to `TorProfileServiceFactory::IsTorDisabled`?

Reply:
Looks like it can